### PR TITLE
Add some adjectives to the namesgenerator

### PR DESCRIPTION
**Pull request by thaJeztah**
Because .. `hardcore_hamilton` and `inspiring_murdock` 

https://twitter.com/swiftonsecurity/status/801195049165799424

/cc @mstanleyjones

### CODE REVIEW

**Comment by mstanleyjones** [comment]
How about `competent`, `prepared`, `thankful`, `dreamy`, `chilly`, `festive`, `frosty`, `sparkly`, `cold` as some more thematic and seasonal adjectives? I didn't check to see if they were there.

**Comment by justincormack** [comment]
I think it would be nice to change more of the adjectives so you could get
the feel of a version from them...

**Comment by thaJeztah** [comment]
I'll add some more tomorrow. Perhaps I'll remove some of the negative ones ("backstabbing" "berserk" came up at some point I think), suggestions welcome for removal as well

ping @jpetazzo

File: pkg/namesgenerator/names-generator.go [hunk 1]
```diff
@@ -43,10 +43,12 @@ var (
 		"goofy",
 		"grave",
 		"happy",
+		"hardcore",
 		"high",
 		"hopeful",
 		"hungry",
 		"infallible",
+		"inspiring",
```
Comment (vdemeester) [inline pkg/namesgenerator/names-generator.go]: hum I am surprised we didn't have this one already :sweat_smile:

**Comment by thaJeztah** [comment]
Added / replaced some more PTAL

**Comment by jpetazzo** [comment]
I like the positive ones _a lot_!

**Review by justincormack — APPROVED** [review]

File: pkg/namesgenerator/names-generator.go [hunk 2]
```diff
@@ -15,68 +15,90 @@ var (
 		"amazing",
 		"angry",
 		"awesome",
-		"backstabbing",
-		"berserk",
-		"big",
```
Comment (duglin) [inline pkg/namesgenerator/names-generator.go]: what's wrong with "big" and "small" ?

Comment (thaJeztah) [inline pkg/namesgenerator/names-generator.go]: Combined with names they can be considered referring to body size, I thought that could possibly be offending.

Comment (duglin) [inline pkg/namesgenerator/names-generator.go]: :face_with_head_bandage:

**Comment by justincormack** [comment]
@thaJeztah you could add @mstanleyjones "festive" suggestion too...

**Comment by thaJeztah** [comment]
@justincormack done!

**Comment by LK4D4** [comment]
LGTM

**Pull request merged**